src/components/data-grid/FooterPageSize.tsx

```diff
@@ -2,9 +2,9 @@ import React, { useEffect, useState } from 'react';
 import { FooterRowCountProps } from './types';
 import { StyledFooterFooterPageSize } from './styles';
 import { Select } from '../select/Select';
+import ParagraphSmall from '../typography/ParagraphSmall';
```

Comment:
Prefer named imports like it used to be

`import { ParagraphSmall } from '../typography';`

Reply:
Fixed